### PR TITLE
Return client key for ancillary component

### PR DESCRIPTION
**Pull request by JamesWair**
OfferRequest will now return the `client_key` for use in the ancillaries component.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ The format is based on [Keep a Changelog] and this project adheres to [Semantic
   [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
   [Semantic Versioning]: http://semver.org/spec/v2.0.0.html
 
+## [0.6.1] - 2023-09-29
```
Comment (nlopes) [inline CHANGELOG.md:10]: We tend to do this on a different PR - as we also have to bump in `setup.py`

**Review by nlopes — CHANGES_REQUESTED** [review]
Just one minor thing we shouldn't push as part of this PR. All else looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (JamesWair) [reply CHANGELOG.md:10]: Ah ok, will remove 👍 I did also update setup.py to 0.6.1, should I also undo that change?

Comment (nlopes) [reply CHANGELOG.md:10]: Yup.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,15 +8,6 @@ The format is based on [Keep a Changelog] and this project adheres to [Semantic
   [Semantic Versioning]: http://semver.org/spec/v2.0.0.html
 
 
-## [0.6.1] - 2023-09-16
```
Comment (nlopes) [inline CHANGELOG.md:11]: I think by mistake during rebase you removed this - sorry - it was part of a PR I merged only today :( I should have linked with what you were doing here.

Comment (JamesWair) [reply CHANGELOG.md:11]: Ahh I didn't notice this in resolving the conflicts, will replace

**Review by nlopes — APPROVED** [review]
Make sure to squash & merge 🙈
